data_pipeline/convertResourcesScript.ts

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env ts-node
+
+/**
+ * Convert a JSON dump of charities in strapped format to resources in upswyng format
+ *
+ * Useage:
+ * - Convert file to executeable
+ * chmod u+x convertResources.ts
```

Comment:
What is the reason for this usage vs adding as an `npm script` like in our past data_pipeline utilities?